src/components/BreadCrumb/BreadCrumb.module.scss

```diff
@@ -0,0 +1,7 @@
+/* BreadCrumb.module.scss */
+.breadcrumb {
+  display: flex; /* Arrange items inline */
```

Comment:
is it possible to use tailwind-css styles here or directly to Breadcrumb.tsx? the .scss file is useful for the dynamic styling.